app/views/sign_up/completions/show.html.erb

```diff
@@ -24,6 +24,15 @@
         sp: content_tag(:strong, decorated_session.sp_name),
       ) %>
 </p>
+<% if !@multiple_factors_enabled && IdentityConfig.store.select_multiple_mfa_options %>
+  <%= render(AlertComponent.new(type: :warning, class: 'margin-bottom-4')) do %>
+    <%= link_to(
+          t('mfa.second_method_warning.link'),
+          login_two_factor_options_url,
+        ) %>
```

Comment:
Is this link supposed to be functional right now? When I tested, it looks like it just refreshes the same page.

Reply:
It is supposed to be functional. Turns out I used `login_two_factor_options_url` (which doesn't exist) and that caused the 302 that happened in between the click -> redirect -> same page.

Reply:
Hm, after testing again with the latest version of the branch, it still doesn't seem to be working for me.

On a related note, it might also be good to have some feature specs for this flow to have some end-to-end assurance that the user experience is working as expected.

Reply:
I had not pushed the change up yet. That's my mistake.

I agree with having feature specs. I hadn't thought of starting that because most of the tickets for the functionality were about building out the UI as the routing was being completed. It definitely should be done though.

Reply:
I also just learned that those tests are being written in the "good path" flow, that is already being started.